src/components/AttachmentModal.js

```diff
@@ -81,6 +85,25 @@ class AttachmentModal extends PureComponent {
         this.setState({isModalOpen: false});
     }
 
+    /**
+     * Close the confirm modal.
+     */
+    closeConfirmModal() {
+        this.setState({isConfirmModalOpen: false});
+    }
+
+    /**
+     * Check if the attachment size is less than the API size limit.
+     * @param {Object} file
+     * @returns {Boolean}
+     */
+    isValidSize(file) {
+        if (!file) {
```

Comment:
You could tidy this up to a one liner
```
return !file || parseInt(file.size, 10) < CONST.API_MAX_ATTACHMENT_SIZE;
```

Reply:
Sounds good.
We should use `lodashGet` right?
 
`return !file || lodashGet(file, 'size', 0) < CONST.API_MAX_ATTACHMENT_SIZE;`